crates/optimism/cli/src/commands/build_pipeline.rs

```diff
@@ -0,0 +1,102 @@
+use alloy_primitives::B256;
+use futures_util::{Stream, StreamExt};
+use reth_config::Config;
+use reth_consensus::Consensus;
+use reth_db_api::database::Database;
+use reth_downloaders::{
+    bodies::bodies::BodiesDownloaderBuilder, file_client::FileClient,
+    headers::reverse_headers::ReverseHeadersDownloaderBuilder,
+};
+use reth_errors::ProviderError;
+use reth_evm_optimism::OpExecutorProvider;
+use reth_network_p2p::{
+    bodies::downloader::BodyDownloader,
+    headers::downloader::{HeaderDownloader, SyncTarget},
+};
+use reth_node_events::node::NodeEvent;
+use reth_provider::{BlockNumReader, ChainSpecProvider, HeaderProvider, ProviderFactory};
+use reth_prune::PruneModes;
+use reth_stages::{sets::DefaultStages, Pipeline, StageSet};
+use reth_stages_types::StageId;
+use reth_static_file::StaticFileProducer;
+use std::sync::Arc;
+use tokio::sync::watch;
+
+macro_rules! block_executor {
+    ($chain_spec:expr) => {
+        OpExecutorProvider::optimism($chain_spec)
+    };
+}
```

Comment:
this we can remove now and replace with `OpExecutorProvider::optimism($chain_spec)`